src/js/server/hcapi/applications.js

```diff
@@ -4,6 +4,23 @@ const { createHttpError } = require('./errors');
 
 const applicationsRouter = new Router();
 
+applicationsRouter.get('/', (req, res, next) => {
+  HackerApplication
```

Comment:
Yeah I think some kind of computed hacker object is warranted here which contains useful hacker info (university, name, gender etc...) and also useful application info (status, score)